packages/keyboard-shortcuts/src/hooks/use-shortcut.js

```diff
@@ -0,0 +1,85 @@
+/**
+ * External dependencies
+ */
+import Mousetrap from 'mousetrap';
+import 'mousetrap/plugins/global-bind/mousetrap-global-bind';
+import { includes } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { useSelect } from '@wordpress/data';
+import { rawShortcut } from '@wordpress/keycodes';
+import { useEffect } from '@wordpress/element';
+
+/**
+ * Return true if platform is MacOS.
+ *
+ * @param {Object} _window   window object by default; used for DI testing.
+ *
+ * @return {boolean}         True if MacOS; false otherwise.
+ */
+function isAppleOS( _window = window ) {
```

Comment:
What's the point of the argument? It's not like we avoid referencing the window global anyways.

Reply:
Probably, It's not new either, this is just copied for the KeyboardShortcuts implementation which could be refactored later to rely on a the same underlying hook.